media-sound/spotify/spotify-1.1.99.ebuild

```diff
@@ -0,0 +1,133 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop optfeature multiprocessing pax-utils xdg
+
+DESCRIPTION="Spotify is a social music platform"
+HOMEPAGE="https://www.spotify.com/download/linux/"
+
+# Pipe the command below in jq and you can see the available versions and their download urls
+# curl -H "Snap-Device-Series: 16" https://api.snapcraft.io/v2/snaps/info/spotify
+
+# Hopefully helpfup jq expression to parse version, revision and the arch its for
+# jq '.["channel-map"][] | {"version": .version, "arch": .channel.architecture, "revision": .revision}'
+
+SNAP_ID="pOBIoZ2LrCB3rDohMxoYGnbN14EHOgD7"
+AMD64_REVISION="62"
+SRC_URI="
+	amd64? ( https://api.snapcraft.io/api/v1/snaps/download/${SNAP_ID}_${AMD64_REVISION}.snap -> ${P}.snap )
+"
```

Comment:
As it is a single line you don't really need to use more than one line and, as it build only for x86_64, I think you can drop the amd64? `SRC_URI="https://api.snapcraft.io/api/v1/snaps/download/${SNAP_ID}_${AMD64_REVISION}.snap -> ${P}.snap"`

Reply:
Yes, it could I overbuilt it initially while playing around with the snapd api. But keeping it doesnt hurt because you never know if a spotify dev decides to start arm64 builds etc. But im not opposed to removing it.

Reply:
I still hope they will but not that sure